core/src/main/java/com/flipkart/varadhi/core/cluster/ComponentKind.java

```diff
@@ -1,4 +1,4 @@
-package com.flipkart.varadhi.components;
+package com.flipkart.varadhi.core.cluster;
```

Comment:
should ALL be part of KIND?
or ALL should implicitly be represented by including actual kinds.

Reply:
it should be later.. i.e. no ALL.